evergreen.yml

```diff
@@ -173,25 +173,6 @@ functions:
           ./resources/run_adf.sh stop
           exit $EXITCODE
 
-  "run unit-test":
```

Comment:
I guess we can't test if the functions are not shared, is that correct?

Reply:
That's right, it's not visible in the other files.  Their unit testing example has shared functions, but it must be more strict for certification. 
https://github.com/microsoft/DataConnectors/blob/master/samples/UnitTesting/UnitTesting.pq